src/seedu/addressbook/ui/TextUi.java

```diff
@@ -16,21 +10,23 @@
 import seedu.addressbook.commands.CommandResult;
 import seedu.addressbook.data.person.ReadOnlyPerson;
 
+import static seedu.addressbook.common.Messages.*;
+
 /**
  * Text UI of the application.
  */
 public class TextUi {
 
     /** A decorative prefix added to the beginning of lines printed by AddressBook */
-    private static final String LINE_PREFIX = "|| ";
+    public static final String LINE_PREFIX = "|| ";
 
     /** A platform independent line separator. */
-    private static final String LS = System.lineSeparator();
+    public static final String LS = System.lineSeparator();
 
-    private static final String DIVIDER = "===================================================";
+    public static final String DIVIDER = "===================================================";
```

Comment:
While implementing SRP, please try to abstract out all the information while creating a new class. Eg. DIVIDER, LINE_PREFIX, etc., variables can be defined inside Formatter and TextUi doesn't need to know about them.